k2/python/k2/rnnt_loss.py

```diff
@@ -229,7 +256,7 @@ def get_rnnt_logprobs_joint(
     This function is called from rnnt_loss().
 
     Args:
-      joint:
+      logits:
         The output of joiner network, with shape (B, T, S + 1, C),
         i.e. batch, time_seq_len, symbol_seq_len+1, num_classes
```

Comment:
I think we should remove "possibly including EOS", since RNN-T does not use EOS.